config/gocdb_schema.xml

```diff
@@ -266,6 +266,9 @@
 	    <length>255</length>
 	    <regex>/^(\+?[0-9\-\.\s\)\(]+)?$/</regex>
 	</field>
+	<field>
+		<fname>UNLINK_HOMESITE</fname>
+	</field>
```

Comment:
It's not immediately clear to me why we need to add something to the schema in order to remove something. I could probably benefit from some additional comments explaining how the different (non-view) parts of the PR fit together.